src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/adjudications/ReportedAdjudicationDto.kt

```diff
@@ -0,0 +1,141 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.adjudications
```

Comment:
Across the codebase we have stuck to one data class per model file. This does not stick with the behaviour in the codebase having so many data classes in one file. I'll still approve this PR as it's not inherently a bad way to do this in my opinion but I think it has raised the question that we need to adopt a standard potentially for this as I think the codebase should be consistent.